Xamarin.Forms.Platform.iOS/PageLifecycleManager.cs

```diff
@@ -69,25 +68,17 @@ public void Dispose()
 
 		public void HandlePageAppearing()
 		{
-			if (_appeared)
```

Comment:
Page.cs already has these checks

https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Core/Page.cs#L421

Having the logic inside Page determine whether it's in a valid state to appear or not also keeps the logic in the xplat.

For example with Shell the Page needs to figure out if it's the visible ShellContent. Putting that logic inside this manager I think would be excessive.

I also feel like this class should be alerting our App.cs that an appearance has happened and then the xplat figures out where to route that appearing action. The xplat code knows what page is visible. Having one of these per Page doesn't seem necessary because the PageController here isn't playing any roll with the observer, since the observer is a global event.